docs/getting-started/local-build.md

```diff
@@ -69,6 +69,13 @@ Run `make` to build a combined frontend and backend binary. The frontend is conf
 make
 ```
 
+:::info
+If you receive an error message:
+```
+tools/preflight-checks.sh: line 12: realpath: command not found
+```
+Then `realpath` might not be in your `PATH` (you can check via `which realpath`) nor linked via `brew`.  To resolve this error message on OSX, try running `brew link coreutils` first, before running `make` again.
```

Comment:
The first sentence reads like its osx specific as it calls out brew. I think we can just nix the `nor linked via brew` part. The second part of this info clarifies for users on osx.

Reply:
Thanks @mikecutalo — I credited you at at https://github.com/lyft/clutch/pull/2558/commits/e79f1a8e2ff1c6f608061c0763c4866cc2847ddf